bundles/c/c_lexer.moon

```diff
@@ -41,13 +48,15 @@ howl.aux.lpeg_lexer ->
     sequence {
       c('keyword', word { 'class', 'struct' })
       ws^1
-      c('type_def', ident)
+      attribute_spec^0
+      c('type_def', ident) * (c('operator', P'::') + c('type_def', ident))^0
       ws^0
+      (((V'all' + P 1) - S':{}=;') + ws)^0
```

Comment:
If this is going to be referencing grammar rules I'd rather have it in the grammar itself. That leaves anything outside of the grammar as "simple" helpers.